tests/unit/esgf/test_search.py

```diff
@@ -208,7 +208,7 @@ def test_esgf_search_files(mocker):
     files = _search.esgf_search_files(facets)
 
     SearchConnection.assert_called_once_with(
-        url='https://esgf-index1.example.com/esg-search')
+        url='https://esgf.ceda.ac.uk/esg-search')
```

Comment:
This looks like a mistake?

Reply:
:man_facepalming:

Reply:
It's still different from what it was, can we just keep it the same?

Reply:
why keep the `index1` if that has completely vanished from the internets?

Reply:
There are two index nodes in the mock search connection configuration used for testing in this file. Since these index nodes aren't real anyway (they use the example.com domain), I don't understand why this needs to be changed. But if you're really keen on changing them anyway, then please change both (now you're only changing the first one) and keep the numbering in the names because it makes the test code easier to understand if there is a '1' in the name of the first one and a '2' in the name of the second one.

Reply:
It looks like we're misunderstanding each other. I wrote a new comment below, is that any more clear?